_content/tour/por/variables.article

```diff
@@ -65,8 +66,8 @@ performance.
     String             "" (empty)
     Pointer            nil
 
-** Declare and Initialize
-The keyword var can be used to construct values to their zero value state for all types.
+** Declarar e Inicializar
```

Comment:
@rschio I'm wondering if we should use "Declaração e Inicialização". What do you think? But anyway I'm happy witht the current.

Reply:
Yeah I think "Declaração e Inicialização" fits better, I will fix.